app/views/admin/terms_of_service/_links.html.haml

```diff
@@ -0,0 +1,6 @@
+.content__heading__tabs
+  = render_navigation renderer: :links do |primary|
+    :ruby
+      primary.item :current, safe_join([material_symbol('description'), t('admin.terms_of_service.current')]), admin_terms_of_service_index_path
```

Comment:
Is this icon intentionally the same as the next one, or still WIP?

Reply:
Intentionally, since they're the same kind of text document.